include/ignition/sensors/ImuSensor.hh

```diff
@@ -135,6 +135,12 @@ namespace ignition
       /// \brief Get the gravity vector
       /// \return Gravity vectory in meters per second squared.
       public: math::Vector3d Gravity() const;
+      
+      /// \brief Convey worl frame's orientation and heading offset
+      /// \param[in] _rot heading offset
+      /// \param[in] _relativeTo world frame orintation, "ENU" by default
+      public: void SetWorldFrameOrientation(
+        const math::Quaterniond &_rot, std::string _relativeTo);
```

Comment:
Please create an enum instead of passing a string for the frame values. See [NoiseType](https://github.com/ignitionrobotics/ign-sensors/blob/ign-sensors6/include/ignition/sensors/Noise.hh#L65-L71) for an example

Reply:
Done !